tests/preprocessing/test_bcdi_utils.py

```diff
@@ -80,8 +80,7 @@ def test_bin_roi(self):
         peak = find_bragg(
             data=self.data, peak_method="max", roi=(3, 3, 4, 2), binning=(3, 3, 4)
```

Comment:
the tests relative to `find_bragg` are also not in the scope of this pull request.